src/ColorHeatmap.js

```diff
@@ -74,7 +74,7 @@ export default class ColorHeatmap extends React.Component {
      */
     colorScale: PropTypes.func,
     /**
-     * ...or else one will be constructed from colors, colorStops and interpolator
+     * ...or else one will be constructed from colors, valueDomain and interpolator
```

Comment:
please run `npm docs` so the docs build also gets updated when we merge this in

Reply:
@krissalvador27 done